dev/fattest.simplicity/src/componenttest/topology/database/container/DatabaseContainerType.java

```diff
@@ -37,25 +38,27 @@
  */
 @SuppressWarnings("rawtypes")
 public enum DatabaseContainerType {
-    DB2("jcc.jar", Db2Container.class.getCanonicalName(), Properties_db2_jcc.class, //
+    DB2("jcc.jar", "", Db2Container.class.getCanonicalName(), Properties_db2_jcc.class, //
         DockerImageName.parse("kyleaure/db2:1.0").asCompatibleSubstituteFor("ibmcom/db2")),
-    Derby("derby.jar", DerbyNoopContainer.class.getCanonicalName(), Properties_derby_embedded.class, DockerImageName.parse("")),
-    DerbyClient("derbyclient.jar", DerbyClientContainer.class.getCanonicalName(), Properties_derby_client.class, DockerImageName.parse("")),
-    Oracle("ojdbc8_g.jar", OracleContainer.class.getCanonicalName(), Properties_oracle.class, //
+    Derby("derby.jar", "", DerbyNoopContainer.class.getCanonicalName(), Properties_derby_embedded.class, DockerImageName.parse("")),
+    DerbyClient("derbyclient.jar", "", DerbyClientContainer.class.getCanonicalName(), Properties_derby_client.class, DockerImageName.parse("")),
+    Oracle("ojdbc8_g.jar", "ojdbc8.jar", OracleContainer.class.getCanonicalName(), Properties_oracle.class, //
```

Comment:
Why do you have the old `ojdbc8_g.jar` driver listed as the primary one and the new one as the alternate one?  Are you going to switch the places of the `ojdbc8_g.jar` and `ojdbc8.jar` in the method arguments when you remove the temporary solution?

Reply:
This temporary code is so we can make updates to our internal WebSphere Liberty project.